src/Task/TestFramework/PhpUnitTask.php

```diff
@@ -212,11 +238,11 @@ protected function runPhpUnit(): void {
         '--debug',
         "--configuration={$this->fixture->getPath('docroot/core/phpunit.xml.dist')}",
         '--exclude-group=orca_ignore',
+        '--testsuite=sut',
```

Comment:
Because this is added unconditionally, it will also limit integrated tests to the SUT, so non-SUT tests will never be run.